app.js

```diff
@@ -54,6 +54,25 @@ app.use(async (ctx, next) => {
     ctx.near = await nearPromise;
     await next();
 });
+const SECURITY_CODE_DIGITS = 6;
+const NEW_ACCOUNT_AMOUNT = process.env.NEW_ACCOUNT_AMOUNT;
+const password = require('secure-random-password');
+const models = require('./models');
+/********************************
+Routes
```

Comment:
nit: not clear what this comment tries to explain. Stuff directly below isn't "Routes"